genIcons.js

```diff
@@ -0,0 +1,8 @@
+const { execSync } = require('child_process')
+
+const arg = process.argv[2] || ''
+const icon = `icons/${arg}`
+execSync(
+  `npx @svgr/cli --icon -d renderer/components/Icon ${icon} && npm run lint-fix -- renderer/components/Icon`,
```

Comment:
Can you update the lint command so that it only runs against the file in question too?

Reply:
good catch